src/shims/unix/fs.rs

```diff
@@ -332,7 +337,9 @@ impl FileHandler {
 }
 
 impl<'mir, 'tcx: 'mir> EvalContextExtPrivate<'mir, 'tcx> for crate::MiriInterpCx<'mir, 'tcx> {}
-trait EvalContextExtPrivate<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
+pub(super) trait EvalContextExtPrivate<'mir, 'tcx: 'mir>:
```

Comment:
Nono, the entire point of the `Private` trait is for it to be private. Don't change that.

Reply:
I made the trait public in order to use `handle_not_found`. I moved that function out to `EvalContextExt` and made the traits private again, in a single commit.